### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by AsciiWolf**

### CODE REVIEW

File: src/permissionEntryRow.ui [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="halign">start</property>
-                <property name="label" translatable="yes">Pemission</property>
+                <property name="label" translatable="yes">Permission</property>
```
Comment (tchx84) [inline src/permissionEntryRow.ui]: Thanks for catching those. Note that these strings are not user-facing, so additionally we must remove these "translatable" tags, so the string can be remove (automatically) from the .pot and .po files when I updated them.

File: po/es.po [hunk 2]
```diff
@@ -57,7 +57,7 @@ msgid "Description"
 msgstr "Descripción"
 
 #: src/permissionEntryRow.ui:29 src/permissionSwitchRow.ui:29
-msgid "Pemission"
+msgid "Permission"
```
Comment (tchx84) [inline po/es.po]: No need to fix it manually here.

File: po/flatseal.pot [hunk 3]
```diff
@@ -55,7 +55,7 @@ msgid "Description"
 msgstr ""
 
 #: src/permissionEntryRow.ui:29 src/permissionSwitchRow.ui:29
-msgid "Pemission"
+msgid "Permission"
```
Comment (tchx84) [inline po/flatseal.pot]: Same here. No need for this.

File: src/permissionSwitchRow.ui [hunk 4]
```diff
@@ -26,7 +26,7 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="halign">start</property>
-                <property name="label" translatable="yes">Pemission</property>
+                <property name="label" translatable="yes">Permission</property>
```
Comment (tchx84) [inline src/permissionSwitchRow.ui]: Like mentioned above, additionally, remove the translateable tag.

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: src/permissionEntryRow.ui (hunk 1 above)
Comment (AsciiWolf) [reply src/permissionEntryRow.ui]: Done. :-)

**Pull request merged**